src/main/fc/config.c

```diff
@@ -429,6 +429,8 @@ static void activateConfig(void)
     activateControlRateConfig();
 
     resetAdjustmentStates();
+    
+    useRcControlsConfig();
```

Comment:
Well, I think we should avoid using the term "config". How about `rcUpdateModeActivationConditions`?